src/components/Search/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ChangeEvent } from 'react';
+import { getCoinsList } from '../../api/api';
+import { Coin } from '../../types/types';
+import SearchIcon from '../icons/SearchIcon';
+import styles from './Search.module.scss';
+import Loader from '../Loader';
+
+const { search, search__input, search__submit } = styles;
+
+type SearchState = {
+  searchValue: string;
+  isLoading: boolean;
+};
+
+type SearchProps = {
+  updatedCoinsList: (loadedCoins: Coin[]) => void;
+};
```

Comment:
I suggest creating a separate file within this folder and moving the types into this file